readthedocs/builds/views.py

```diff
@@ -82,6 +84,30 @@ class BuildDetail(BuildBase, DetailView):
     def get_context_data(self, **kwargs):
         context = super(BuildDetail, self).get_context_data(**kwargs)
         context['project'] = self.project
+        scheme = ("https://github.com/rtfd/readthedocs.org/issues/new"
+                  "?title={title}{build_id}"
+                  "&body={body}")
+
+        body = ("# Details:\n\n"
+                "*Project URL: https://readthedocs.org/projects/{projname}/\n"
+                "*Build URL(if applicable): https://readthedocs.org{build_path}\n"
+                "*Read the Docs username(if applicable): {uname}\n\n"
+                "## Expected Result\n\n"
+                "*A description of what you wanted to happen*\n\n"
+                "## Actual Result\n\n"
+                "*A description of what actually happened*")
+
+        body_dict = {'projname': self.project,
+                     'build_path': self.request.path,
+                     'uname': self.request.user}
+
+        scheme_dict = {'title': quote("Build error with build id #"),
+                       'build_id': self.request.path[-2:-1],
```

Comment:
There is a better way to get the `build_id`? The id isn't always 2 characters length.

Reply:
Fixed this!